install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -175,6 +175,14 @@ data:
         - name: ISTIO_META_NETWORK
           value: "{{ .Values.global.network }}"
         {{- end }}
+        - name: ISTIO_META_WORKLOAD_SDS_ENABLED
```

Comment:
How about: 

if {{ sds.enabled }}
.....
end

- this way if sds is not enabled we don't spam everyone with extra meta ( it's sent around quite a lot ).

Reply:
done, thanks